examples/get_meter_reading.py

```diff
@@ -19,9 +27,37 @@
 
 
 async def main():
+
+    caiKey = 'CA Issuers - URI:'
+    reIssuersURI = re.compile(r"(https?://+[\w\d:#@%/;$()~_?\+-=\\\.&]*)", re.UNICODE)
+
+    caIssuersURI = None
+    context =  ssl.create_default_context(capath=certifi.where())
+    context.check_hostname = False
+    context.verify_mode = ssl.CERT_NONE
+    with context.wrap_socket(socket.socket(), server_hostname=BASE_HOSTNAME) as s:
+        s.connect((BASE_HOSTNAME, 443))
+        cert_bin = s.getpeercert(True)
+        x509 = crypto.load_certificate(crypto.FILETYPE_ASN1, cert_bin)
+        for idx in range(x509.get_extension_count()):
+            ext = x509.get_extension(idx)
+            short_name = ext.get_short_name()
+            if short_name == b"authorityInfoAccess":
+                authorityInfoAccess = str(ext)
+
+                caiIndx = authorityInfoAccess.find(caiKey)
+                if (caiIndx > -1):
+                    caiValue = authorityInfoAccess[caiIndx:]
+                    caIssuersURI = reIssuersURI.findall(caiValue)[0]
+
+    if (caIssuersURI != None):
+        with urllib.request.urlopen(caIssuersURI) as certReq:
+            certData = certReq.read()
+            context.load_verify_locations(cafile=certifi.where(), cadata = certData)
+
```

Comment:
Can this be moved to another file and imported here? I’d like to keep the examples simple if possible.